application/grifts/import.go

```diff
@@ -1086,3 +1090,82 @@ func fixedPointStringToInt(s, desc string) int {
 	fractionalPart := stringToInt(parts[1], desc+" right of decimal")
 	return intPart*100 + fractionalPart
 }
+
+var states = map[string]string{
+	"Alabama":              "AL",
+	"Alaska":               "AK",
+	"Arizona":              "AZ",
+	"Arkansas":             "AR",
+	"California":           "CA",
+	"Colorado":             "CO",
+	"Connecticut":          "CT",
+	"Delaware":             "DE",
+	"District of Columbia": "DC",
+	"Florida":              "FL",
+	"Georgia":              "GA",
+	"Hawaii":               "HI",
+	"Idaho":                "ID",
+	"Illinois":             "IL",
+	"Indiana":              "IN",
+	"Iowa":                 "IA",
+	"Kansas":               "KS",
+	"Kentucky":             "KY",
+	"Louisiana":            "LA",
+	"Maine":                "ME",
+	"Montana":              "MT",
+	"Nebraska":             "NE",
+	"Nevada":               "NV",
+	"New Hampshire":        "NH",
+	"New Jersey":           "NJ",
+	"New Mexico":           "NM",
+	"New York":             "NY",
+	"North Carolina":       "NC",
+	"North Dakota":         "ND",
+	"Ohio":                 "OH",
+	"Oklahoma":             "OK",
+	"Oregon":               "OR",
+	"Maryland":             "MD",
+	"Massachusetts":        "MA",
+	"Michigan":             "MI",
+	"Minnesota":            "MN",
+	"Mississippi":          "MS",
+	"Missouri":             "MO",
+	"Pennsylvania":         "PA",
+	"Rhode Island":         "RI",
+	"South Carolina":       "SC",
+	"South Dakota":         "SD",
+	"Tennessee":            "TN",
+	"Texas":                "TX",
+	"Utah":                 "UT",
+	"Vermont":              "VT",
+	"Virginia":             "VA",
+	"Washington":           "WA",
+	"West Virginia":        "WV",
+	"Wisconsin":            "WI",
+	"Wyoming":              "WY",
+	"Alberta":              "AB",
+}
+
+func getState(c string) string {
+	c = trim(c)
+
+	if state, ok := states[c]; ok {
+		return state
+	}
+	if len(c) == 2 {
+		return c
+	}
+	return ""
+}
+
+func getCountry(c string) string {
+	c = trim(c)
+
+	if _, ok := states[c]; ok {
```

Comment:
Could simplify it some by combining the if statements, but either way is fine with me.

```suggestion
	if _, ok := states[c]; ok || len(c) <= 2 {
		return ""	
	}
	
	return c
```